frontend/src/global_styles/content/_toast.sass

```diff
@@ -145,31 +144,38 @@ $nm-upload-box-padding: rem-calc(15) rem-calc(25)
   &.-error
     @extend %error-placeholder
 
-    &::before
+    &:not(.toast-icon)::before
       @include messages-icon
       @extend %nm-icon-error
 
   &.-success
     @extend %success-placeholder
 
-    &::before
+    &:not(.toast-icon)::before
       @include messages-icon
       @extend %nm-icon-success
 
   &.-warning
     @extend %warning-placeholder
 
-    &::before
+    &:not(.toast-icon)::before
       @include messages-icon
       @extend %nm-icon-warning
 
   &.-info
     @extend %info-placeholder
 
     &::before
+      color: $nm-color-info-icon
```

Comment:
Similar to how you extracted the color here, you have to do the same thing for the other types (error, success, warning) as well. Otherwise, the icon will be black there:

<img width="654" alt="Bildschirmfoto 2024-04-15 um 15 09 18" src="https://github.com/opf/openproject/assets/7457313/637d40ae-a60b-461f-8c19-485b7e816fd6">